components/settings/ConfigureTranslation.tsx

```
@@ -0,0 +1,437 @@
+              </Pressable>
+            );
+          })}
+        </View>
+
+        {/* Input Transcription */}
+        <Text style={styles.sectionLabel}>Input Transcription</Text>
+        <View style={styles.optionCard}>
+          <View style={styles.optionCopy}>
+            <Text style={styles.optionTitle}>Transcribe Source Audio</Text>
+            <Text style={styles.optionSubtitle}>
+              Receive source-language text transcripts of the speaker's audio.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)